packit/cli/utils.py

```diff
@@ -369,3 +370,14 @@ def get_existing_config(working_dir: Path) -> Optional[Path]:
         if config_file_path.is_file():
             return config_file_path
     return None
+
+
+def get_precommit_config(working_dir: Path) -> Optional[Path]:
+    config_file_path = working_dir / ".pre-commit-config.yaml"
+    if config_file_path.is_file():
+        return config_file_path
+    return None
```

Comment:
I'll show you why I hate this language (JS people love this shit too):

```suggestion
    return config_file_path.is_file() and config_file_path
```

(and yes, as long as you doesn't check for `is None` in the other module, this really does work :D)

Don't you dare to commit it :D